### DISCUSSION TITLE
Toggle hidden apps in apps drawer

### DISCUSSION THREAD

**Discussion by ghost** [tanujnotes/Olauncher#179] [Ideas]
An icon, hide/show, in the search field in the app drawer to toggle show/hide hidden apps. 

Since there's already the show all apps used when adding an app to the home screen this toggle in the app drawer shouldn't add any weight. 😀 Just makes it easier for the user to access a hidden app when needed.

Thanks!

**Comment by NaimMasum** [comment]
Is there any way to unhide apps

**Reply by ghost** [reply]
Go into settings, tap on Olauncher, that brings up all hidden apps, long press on the one you want to unhide...

**Reply by NaimMasum** [reply]
Thanks

**Comment by tanujnotes** [comment]
This defeats the whole purpose of hidden apps.

**Comment by helpdeskdan** [comment]
It would really cool if this setting could be locked with pin and/or biometric.  I would pay for that.
